wherehows-web/app/routes/datasets/dataset.js

```diff
@@ -319,6 +323,33 @@ export default Route.extend({
         return Promise.reject(new Error('Dataset references request failed.'));
       })
       .catch(() => set(controller, 'hasReferences', false));
+
+    // TODO: Get stutus current user to see data in ACL access tab
+    Promise.resolve(currentUser())
+      .then(userInfo => {
+        setProperties(controller, {
+          userInfo
+        });
+        return checkAclAccess(userInfo.userName).then(value => {
+          setProperties(controller, {
+            aclAccessResponse: value,
+            currentUserInfo: userInfo.userName,
+            aclUsers: value.body
+          });
+        });
+      })
+      // return Promise.resolve(checkAclAccess(userInfo.userName)).then(value => {
+      //   console.log('value', value);
+      //   setProperties(controller, {
+      //     aclAccessResponse: value,
+      //     currentUserInfo: userInfo.userName,
+      //     aclUsers: value.body
+      //   });
+      // });
+      // })
+      .catch(error => {
+        console.log('It is ACL response error', error);
```

Comment:
We usually don't want to keep console logs in the code we push.

Reply:
agreed